velox/exec/fuzzer/RowNumberFuzzerRunner.h

```diff
@@ -50,23 +48,23 @@
 ///  --enable_oom_injection: randomly trigger OOM while executing query plans.
 /// e.g:
 ///
-///  $ ./velox_row_number_fuzzer_test \
+///  $ ./velox_row_number_fuzzer \
 ///         --seed 123 \
 ///         --duration_sec 600 \
 ///         --v=1
 
-namespace facebook::velox::exec::test {
+namespace facebook::velox::exec {
 
 class RowNumberFuzzerRunner {
  public:
   static int run(
       size_t seed,
-      std::unique_ptr<ReferenceQueryRunner> referenceQueryRunner) {
+      std::unique_ptr<test::ReferenceQueryRunner> referenceQueryRunner) {
     serializer::presto::PrestoVectorSerde::registerVectorSerde();
     filesystems::registerLocalFileSystem();
     rowNumberFuzzer(seed, std::move(referenceQueryRunner));
     return RUN_ALL_TESTS();
```

Comment:
Should we add a main method for the runner and remove this line？

Reply:
Fixed this. The main method already exists.